services/src/main/java/org/keycloak/services/clientregistration/ClientRegistrationAuth.java

```diff
@@ -149,8 +151,8 @@ public RegistrationAuth requireCreate(ClientRegistrationContext context) {
         }
 
         try {
-            ClientRegistrationPolicyManager.triggerBeforeRegister(context, registrationAuth);
-        } catch (ClientRegistrationPolicyException crpe) {
+            ClientPolicyManager.triggerBeforeRegister(context, registrationAuth);
```

Comment:
I couldn't find how the ClientPolicyManager has replaced the ClientRegistrationPolicyManager yet? Can you explain how the old client registration policies are executed now?

Reply:
`ClientPolicyManager` wraps the execution of the old client registration policy `ClientRegistrationPolicyManager`.

In the new client policy `ClientPolicyManager.triggerBeforeRegister()`, the old client registration policy `ClientRegistrationPolicyManager.triggerBeforeRegister()` is called.

Also, in the new client policy `ClientPolicyManager.triggerBeforeUpdate()`, the old client registration policy `ClientRegistrationPolicyManager.triggerBeforeUpdate()` is called.